### ISSUE TITLE
Subfamilies with identical extent

### ISSUE THREAD

**Issue by xflr6** [clld/glottolog#196]
This somewhat advanced `glottolog3` [check for identical families](https://github.com/clld/glottolog3/blob/c0ed353dacfb62cd42bf779c0874878a023dfeed/glottolog3/scripts/check_db_consistency.py#L67-L68) has not yet been ported to the checks in [scripts/treedb.py](https://github.com/clld/glottolog/blob/master/scripts/treedb.py) (which can be run directly with the repo data). We have two violations in glottolog 3.2:
```shell
$ python glottolog3/scripts/check_db_consistency.py development.ini
FamiliesDistinct: 4 invalid
    (violating Each family node has a unique set of member languages.)
    east2792, fare1243, buki1252, puna1280
```
http://glottolog.org/resource/languoid/id/fare1243
http://glottolog.org/resource/languoid/id/buki1252

**Comment by d97hah** [comment]
Both cases are typo-like mistakes by me, will PR shortly

2018-01-25 15:21 GMT+01:00 Sebastian Bank <notifications@github.com>:

**Comment by xflr6** [comment]
Fixed by #199
